modules/deploy-app.adoc

```diff
@@ -30,36 +30,36 @@ From the {product-title} web console, you can deploy a test application from the
 
 . In the *Developer Catalog* dialog, select *All services*.
 
-. In the *Developer Catalog* page, select *Languages* -> *JavaScript* from the menu.
-
-. Click *Node.js*, and then click *Create Application* to open the *Create Source-to-Image Application* page.
-+
-[NOTE]
-====
-You might need to click *Clear All Filters* to display the *Node.js* option.
-====
-
-. In the *Git* section, click *Try Sample*.
-
-. Add a unique name in the *Name* field. The value will be used to name the associated resources.
-
-. Confirm that *Deployment* and *Create a route to the application* are selected.
-
-. Click *Create* to deploy the application. It will take a few minutes for the pods to deploy.
-
-. Optional: Check the status of the pods in the *Topology* pane by selecting your *nodejs* app and reviewing its sidebar. You must wait for the `nodejs` build to complete and for the `nodejs` pod to be in a *Running* state before continuing.
-
-. When the deployment is complete, click route URL for the application, which has a format similar to the following:
-+
-----
-http://nodejs-<project>.<cluster_name>.<hash>.<region>.openshiftapps.com/
-----
-+
-A new tab in your browser opens with a message similar to the following.
-+
-----
-Welcome to your Node.js application on OpenShift
-----
+// . In the *Developer Catalog* page, select *Languages* -> *JavaScript* from the menu.
```

Comment:
@akram 
I have "highlighted" this content by commenting it out so that you could review it. 
Should we remove or update this procedure?

Reply:
this section is related to the use of the nodejs s2i image. So, we can keep it as it is not using the nodejs agent image but rather it is directly using an s2i image.